internal/broker/broker.go

```diff
@@ -482,7 +482,7 @@ func (b *Broker) handleIsAuthenticated(ctx context.Context, session *sessionInfo
 			return AuthRetry, errorMessage{Message: "could not load cached info"}
 		}
 
-		if authInfo.UserInfo.Name == "" {
+		if !session.isOffline {
 			authInfo.UserInfo, err = b.fetchUserInfo(ctx, session, &authInfo)
 			if err != nil {
 				slog.Error(err.Error())
```

Comment:
This has the effect that the we deny authentication when something goes wrong during refresh, even if the local token is still valid. I think it would be better to log the error and proceed with the login in that case.

Reply:
That's intended. If we want to refresh the user information every authentication, we need to fail if something goes wrong. IMO, we should not have this kind of nondeterministic behavior, especially when considering PAM and the user information on the system.